config/defaults.go

```diff
@@ -30,7 +30,8 @@ var (
 	}
 )
 
-func defaultGrpcOptions(t *trace.Driver, secure bool, tlsConfig *tls.Config) (opts []grpc.DialOption) {
+func defaultGrpcOptions(t *trace.Driver, secure bool, tlsConfig *tls.Config) []grpc.DialOption {
+	var opts []grpc.DialOption
```

Comment:
You can create a slice with predefined capacity
Because next line will re-allocate internal slice capacity

Reply:
Alternatively you can make slice from static
Such as
```go
opts := []grpc.DialOption{
		grpc.WithKeepaliveParams(
			DefaultGrpcConnectionPolicy,
		),
		// use round robin balancing policy for fastest dialing
		grpc.WithDefaultServiceConfig(`{
			"loadBalancingPolicy": "round_robin"
		}`),
		// limit size of outgoing and incoming packages
		grpc.WithDefaultCallOptions(
			grpc.MaxCallRecvMsgSize(DefaultGRPCMsgSize),
			grpc.MaxCallSendMsgSize(DefaultGRPCMsgSize),
		),
		// use proxy-resolvers
		// 1) for interpret schemas `ydb`, `grpc` and `grpcs` in node URLs as for dns resolver
		// 2) for observe resolving events
		grpc.WithResolvers(
			xresolver.New("", t),
			xresolver.New("ydb", t),
			xresolver.New("grpc", t),
			xresolver.New("grpcs", t),
		),
}
```

Reply:
I fixed this by adding the make with fixed capacity.